### PR TITLE
Rdm 6716

### PR DESCRIPTION
**Pull request by kiran-yenigala-hmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/ccd-case-ui-toolkit",
-  "version": "2.62.25-RDM-5007-prerelease",
+  "version": "2.62.26-RDM-6716-prerelease",
```
Comment (rafalkalita) [inline package.json]: Is this relevant to the change?

Comment (kiran-yenigala-hmcts) [reply package.json]: I have discussed with Ashley on the versioning... He advised me to use 2.62.26 for this change...
